meilisearch-http/src/option.rs

```diff
@@ -11,10 +11,13 @@ pub struct Opt {
     pub http_addr: String,
 
     /// The master key allowing you to do everything on the server.
-    #[structopt(long, env = "MEILI_API_KEY")]
-    pub api_key: Option<String>,
+    #[structopt(long, env = "MEILI_MASTER_KEY")]
+    pub master_key: Option<String>,
 
-    /// Do not send analytics to Meili.
-    #[structopt(long, env = "MEILI_NO_ANALYTICS")]
-    pub no_analytics: bool,
```

Comment:
Please out this env variable back, the usage is forced to be `--no-analytics=true` no `--no-analytics`.